pilot/pkg/security/authz/model/model.go

```diff
@@ -161,8 +167,8 @@ func NewModel(role *istio_rbac.ServiceRole, bindings []*istio_rbac.ServiceRoleBi
 	return m
 }
 
-// NewModel constructs a Model from v1beta1 Rule.
-func NewModelFromV1beta1(rule *security.Rule) *Model {
+// NewModelV1beta1 constructs a Model from v1beta1 Rule.
+func NewModelFromV1beta1(trustDomain string, trustDomainAliases []string, rule *security.Rule) *Model {
```

Comment:
Also change the function name to `NewModelV1beta1`

Reply:
Done. Thanks!